go/kbfs/libkbfs/keybase_daemon_rpc.go

```diff
@@ -465,6 +465,6 @@ func newNotifyServiceHandler(config Config, log logger.Logger) keybase1.NotifySe
 func (k *KeybaseDaemonRPC) FavoritesChanged(ctx context.Context,
 	uid keybase1.UID) error {
 	k.log.Debug("Received FavoritesChanged RPC.")
-	k.config.KBFSOps().RefreshCachedFavorites(ctx)
+	go k.config.KBFSOps().RefreshCachedFavorites(ctx, true)
```

Comment:
Why does this both need to be in a goroutine, and pass `async = true`?  Seems like one is canceling out the other.  I would think the only reason to block on fetching the favorites list is so that we're guaranteed it's done by the time we return, but the `go` construct means we're not even waiting for it to return.  (I agree that we shouldn't be blocking notifications while refreshing the favorites -- I guess I just don't understand the need for `async = true` here, as it seems useless).